### PR TITLE
QAP-107 Epicentr e2e buying household appliance(checking cart)

### PR DESCRIPTION
**Pull request by YevheniiBond**

### CODE REVIEW

File: src/test/java/YevheniiBondarenko/Tests/Pages/ChoseNewLocation.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    }
+    public WebElement catalogOfProduct(){return $(CATALOG_OF_PRODUCT);}
+    public WebElement householdAppliance(){return $(HOUSEHOLD_APPLIANCE);}
+    public List<WebElement> listOfProduct1(){return $$(LIST_OF_PRODUCT_1);}
+    public WebElement aeroGrill(){return $(AERO_GRILL);}
+    public WebElement myChooiseItem(){return $(MY_ITEM);}
+    public WebElement buyBtn(){return $(BUY_BTN);}
+    public WebElement blueBuyButton(){return $(BLUE_BUY_BUTTON);}
+    public WebElement yellowNumberOnACart(){return $(YELLOW_NUMBER_ON_THE_CART);}
+}
```
[the remark this answers is not in the document]
Comment (Wolmeright) [reply src/test/java/YevheniiBondarenko/Tests/Pages/ChoseNewLocation.java:30]: use CTRL+ALT+L

**Review by Tsibatyuk — APPROVED** [review]

**Review by Safron1ta2 — APPROVED** [review]

**Review by Wolmeright — APPROVED** [review]

File: src/test/java/YevheniiBondarenko/Tests/Pages/ChoseNewLocation.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (YevheniiBond) [reply src/test/java/YevheniiBondarenko/Tests/Pages/ChoseNewLocation.java:55]: +++

File: src/test/java/YevheniiBondarenko/Tests/Pages/EldoradoWebElementsPage.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    public WebElement searchingField(){return $(SEARCH_FIELD);}
+    public WebElement mobilePhones(){return  $(MOBILE_PHONES);}
+    public WebElement brovaryCity(){return $(BROVARY_CITY);}
+    public WebElement cartItemsNumber(){return $(CART_ITEMS_NUMBER);}
+    public List<WebElement> listOfMobilePhones(){return $$(LIST_OF_MOBILE_PHONES);}
+    public List<WebElement> colorOfMobilePhones(){return $$(COLOR_OF_THE_PHONE);}
+    public WebElement buyButton(){return $(BUY_BUTTON);}
+    public List<WebElement> deleteItemsFromCart(){return $$(DELETE_ITEMS_FROM_CART);}
+
+
```
[the remark this answers is not in the document]
Comment (YevheniiBond) [reply src/test/java/YevheniiBondarenko/Tests/Pages/EldoradoWebElementsPage.java:33]: +++

File: src/test/java/YevheniiBondarenko/Tests/Tests/ChoseNewLocationEpicentr.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+import YevheniiBondarenko.Tests.Pages.ItemsInsideTheWebElement;
+import org.openqa.selenium.Keys;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class ChoseNewLocationEpicentr extends TestInit {
+
+    @Test
+    public void choseNewLocation(){
+        ItemsInsideTheWebElement choseNewlocation = new ItemsInsideTheWebElement(driver);
```
[the remark this answers is not in the document]
Comment (YevheniiBond) [reply src/test/java/YevheniiBondarenko/Tests/Tests/ChoseNewLocationEpicentr.java:14]: +++

File: src/test/java/YevheniiBondarenko/Tests/Tests/EldoradoE2EBuyingMobilePhone.java [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+import YevheniiBondarenko.Tests.Pages.EldoradoWebElementsPage;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class EldoradoE2EBuyingMobilePhone extends TestInit {
+
+    @Test
+    public void eldoradoE2EAddAndRemoveFromCart(){
```
[the remark this answers is not in the document]
Comment (YevheniiBond) [reply src/test/java/YevheniiBondarenko/Tests/Tests/EldoradoE2EBuyingMobilePhone.java:13]: Fix

File: src/test/java/YevheniiBondarenko/Tests/Tests/EpicentrItemsInsideTheCart.java [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+import BasePageAndTestInit.TestInit;
+import YevheniiBondarenko.Tests.Pages.ChoseNewLocation;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class EpicentrItemsInsideTheCart extends TestInit {
+
+    @Test
+    public void epicentrItemsInCart(){
+        ChoseNewLocation itemsFromEpicentr = new ChoseNewLocation(driver);
```
[the remark this answers is not in the document]
Comment (YevheniiBond) [reply src/test/java/YevheniiBondarenko/Tests/Tests/EpicentrItemsInsideTheCart.java:13]: Fix

**Review by AndriiMyronenko418 — APPROVED** [review]

**Pull request merged**